src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -401,7 +410,7 @@ private function getReadAccessInfo($object, $property)
 
         if (isset($this->readPropertyCache[$key])) {
             $access = $this->readPropertyCache[$key];
-        } else {
+        } elseif (!$this->cache || !$access = $this->cache->fetch(self::CACHE_PREFIX_READ.$key)) {
```

Comment:
I'm afraid that it wouldn't work for `ACCESS_TYPE_METHOD` case, because constant has value `0`